### ISSUE TITLE
Design considerations for the CMS reserch page

### ISSUE THREAD

**Issue by katilp** [cernopendata/opendata.cern.ch#1208]
Recording discussion with Achim:
In the current http://opendata.cern.ch/research/CMS
we could have more purpose-oriented "boxes" like currently for VMs and Getting Started (top right).
(NB: The title for the VM box should be "Installing VM)
We could add there a new box "Physics use-cases" directing to a new template page (Achim can provide first draft of the contents early 2017)

@tiborsimko could Achim use the wiki area to try out the contents for the new page

**Comment by tiborsimko** [comment]
@katilp Yes, using either wiki or issue body might be enough for start. Does Achim have a GitHub account already, so that we could add him to the organisation and grant him rights?

**Comment by daslerr** [comment]
By use cases, do we mean guidance for researchers on how to use Open Data materials? Or something else?

**Comment by katilp** [comment]
More detailed suggestions from Achim:

[webpageheader.txt](https://github.com/cernopendata/opendata.cern.ch/files/698264/webpageheader.txt)

**Comment by daslerr** [comment]
Thanks, @katilp and Achim. These examples will be very useful. 

I think these examples would mesh quite nicely with the flow proposed in #1218. They've also made me realize that I didn't include any paths between examples and experimental landing pages in that flow, so I'll update it. If you have a moment to take a look at that issue and comment, that would be most appreciated.
